src/util/Enums.js

```diff
@@ -78,4 +79,12 @@ export default {
   CUSTOM_URI_CHALLENGE: 'CUSTOM_URI',
   UNIVERSAL_LINK_CHALLENGE: 'UNIVERSAL_LINK',
   APP_LINK_CHALLENGE: 'APP_LINK',
+
+  // Initial Flows
```

Comment:
Every login experience/pipeline is a flow in widget... Can we add more color/context to these flows that describe how it is different than any other flow..

Reply:
These are just possible values for the `flow` configuration. Added additional comment to explain this